code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -161,6 +162,7 @@
 	applies_material_colour = TRUE
 	applies_material_name = TRUE
 	material_armor_multiplier = 0.8
+	material = /decl/material/solid/leather
```

Comment:
this is already defined 5 lines above pls axe

Reply:
VSCODIUM nuked it give me a minute to fix this pain in the ass